styles/product-listing.scss

```diff
@@ -150,14 +151,20 @@
 }
 
 .sort-container {
+  width: 100%;
+  min-width: 335px;
+  max-width: 350px;
   display: flex;
   align-items: flex-end;
 
   #productDropdownFiltring {
+    width: 100%;
+    min-width: 335px;
     max-width: 350px;
```

Comment:
the sort on the collection page still need to fix
<img width="392" alt="Screen Shot 2023-04-06 at 15 17 36" src="https://user-images.githubusercontent.com/108662497/230423747-e9d988ff-c1c8-418e-ad2f-df59d395074c.png">

Reply:
no in ltr version the arrow should be in the right side